src/NuGet.Services.Validation/Entities/ContentScanType.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.Validation.Entities
+{
+    public enum ContentScanType
+    {
+        Sync = 0,
+        BlockingAsync,
+        NonBlocking
```

Comment:
What is the `NonBlocking` type? I only see uses of `ContentScanType.Sync` and `ContentScanType.BlockingAsync` in the CVS pull request: https://nuget.visualstudio.com/NuGetMicrosoft/_git/NuGet.Internal.Jobs/pullrequest/1769?path=%2Fsrc%2FValidation.CvsSubmitter%2FContentScanOperationService.cs

If `NonBlocking` is unused, should we remove this enum option? Also, please add comments and give each value a specific value. For example:

```suggestion
        /// <summary>
        /// A synchronous content scan. Scan results will be returned in the response to the scan submission.
        /// </summary>
        Sync = 0,
        
         /// <summary>
        /// An asynchronous content scan. The scan submission and scan result lookup are separate requests.
        /// </summary>
        BlockingAsync = 1,
        
        /// <summary>
        /// ???
        /// </summary>
        NonBlocking = 2,
```

Reply:
updated summary for non-blocking type